### DISCUSSION TITLE
Adding template-selection as attachment to each email template

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#515] [Ideas]
We want to add a EME-PDF ticket template to an email template. 
Now we have to manually add the EME PDF template to each event.
We have a lot of event managers working with us, and adding the ticket is often forgotten, which leads to a lot of extra work coming from clients asking for a ticket afterwards.

Is it possible to add an option to attach the EME-PDF ticket to the desired email template by default?

**Comment by liedekef** [comment]
The next version will allow PDF templates to mails that currently can have regular attachments too.

**Reply by JanineKall** [reply]
Oh that would be awesome!!!! You are great.
